src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java

```diff
@@ -78,9 +78,9 @@
     @Auditable(action = AuditingAction.DELETE_COMMENT_FOR_COURSE)
     @DeleteMapping(value = "/api/category/{categoryId}/courses/{courseId}/comments/{courseCommentId}")
     @PreAuthorize(value = "@accessToUrlService.hasAccessToDeleteCommentForCourse(#courseCommentId)")
-    public void deleteComment(@PathVariable Long courseCommentId,@PathVariable Long courseId, @PathVariable Long categoryId) {
+    public ResponseEntity deleteComment(@PathVariable Long courseCommentId, @PathVariable Long courseId, @PathVariable Long categoryId) {
         LOGGER.debug("Deleted comment with id:{} for course with id: {}", courseCommentId, courseId);
         courseCommentService.deleteCommentById(courseCommentId);
+        return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
According to REST Best Practices the body of the DELETE method response should not be empty. You can at least return the id of deleted entity, but please check as I'm not sure.

Reply:
Most sources say that DELETE must return response with a 204 status and an empty body.